Library/Formula/makensis.rb

```diff
@@ -30,10 +43,20 @@ def install
     ENV.libstdcxx if ENV.compiler == :clang
 
     # Don't strip, see https://github.com/Homebrew/homebrew/issues/28718
-    scons "STRIP=0", "makensis"
-    bin.install "build/release/makensis/makensis"
+    scons "STRIP=0", "SKIPUTILS=all", "makensis"
```

Comment:
Why the addition of `"SKIPUTILS=all"` here? Is this taking anything away from `stable` or?